coiote-aws-iot-cloud-formation/createThingType/create_thing_type.py

```diff
@@ -0,0 +1,11 @@
+import boto3
+
+client = boto3.client('iot')
+
+
+def lambda_handler(event, context):
+    thing_type = event['coioteDeviceType'].replace('.', ':')
+    existing_thing_types = client.list_thing_types()['thingTypes']
+    existing_thing_types_names = map(lambda t: t['thingTypeName'], existing_thing_types)
```

Comment:
AFAIR using functional expressions, such as `map` is advised against - use a plain list comprehension here.